observations/obs_converters/GPSPW/work/input.nml

```diff
@@ -1,46 +1,48 @@
 
 &preprocess_nml
-    input_obs_kind_mod_file = '../../../../assimilation_code/modules/observations/DEFAULT_obs_kind_mod.F90',
-   output_obs_kind_mod_file = '../../../../assimilation_code/modules/observations/obs_kind_mod.f90',
-     input_obs_def_mod_file = '../../../../observations/forward_operators/DEFAULT_obs_def_mod.F90',
-    output_obs_def_mod_file = '../../../../observations/forward_operators/obs_def_mod.f90',
+    input_obs_kind_mod_file = '../../../../assimilation_code/modules/observations/DEFAULT_obs_kind_mod.F90'
+   output_obs_kind_mod_file = '../../../../assimilation_code/modules/observations/obs_kind_mod.f90'
+     input_obs_def_mod_file = '../../../../observations/forward_operators/DEFAULT_obs_def_mod.F90'
+    output_obs_def_mod_file = '../../../../observations/forward_operators/obs_def_mod.f90'
    input_files              = '../../../../observations/forward_operators/obs_def_altimeter_mod.f90',
                               '../../../../observations/forward_operators/obs_def_reanalysis_bufr_mod.f90',
                               '../../../../observations/forward_operators/obs_def_metar_mod.f90',
                               '../../../../observations/forward_operators/obs_def_dew_point_mod.f90',
                               '../../../../observations/forward_operators/obs_def_gts_mod.f90',
                               '../../../../observations/forward_operators/obs_def_wind_speed_mod.f90',
-                              '../../../../observations/forward_operators/obs_def_rel_humidity_mod.f90',
- /
+                              '../../../../observations/forward_operators/obs_def_rel_humidity_mod.f90'
+  /
 
 &convert_gpspw_nml
- global_data                 = .true.
- obs_window_hr               = 6
- max_num_obs                 = 1000000
- /
+   global_data    = .true.
+   obs_window_hr  = 6
+   max_num_obs    = 1000000
+  /
 
 &obs_kind_nml
- /
+  /
 
 &location_nml
- /
+  /
 
 &utilities_nml
- module_details = .false.
- /
+   module_details = .false.
+  /
 
 &obs_sequence_nml
-   write_binary_obs_sequence = .false.  
+   write_binary_obs_sequence = .false.
+  /
```

Comment:
This was a syntax problem causing this test to fail for no good reason.
While I was at it, I cleaned up.